contrib/lang/scala/packages.el

```diff
@@ -47,6 +47,66 @@ which require an initialization must be listed explicitly in the list.")
         (kbd "n") 'forward-button
         (kbd "N") 'backward-button)
 
+      (evil-leader/set-key-for-mode 'scala-mode
+        "mri"     'ensime-refactor-inline-local
+        "mrl"     'ensime-refactor-extract-local
+        "mrm"     'ensime-refactor-extract-method
+        "mro"     'ensime-refactor-organize-imports
+        "mrr"     'ensime-refactor-rename
+        "mrt"     'ensime-import-type-at-point
+
```

Comment:
It is not in the conventions for now but the refactoring is planned to be on `SPC m R` instead of `SPC m r`.